examples/ChatServer.py

```diff
@@ -4,8 +4,8 @@
 from time import sleep, localtime
```

Comment:
Same here. Cluttering up the current namespace
```suggestion
from time import sleep, localtime
```

Reply:
I'd love to disagree here, as stated in the Zen of Python:

> Flat is better than nested.

Namespace in Python is to avoid name collision, unlike in e.g. Java.or.many.other.languages.